README.md

```diff
@@ -62,6 +90,20 @@ val encoded: ByteArray // Encoded message that adheres to RFC 8323.
 val message = encoded.decode<Tcp>()
 ```
 
+#### Javascript Decoding Example
+
+Decode a `ByteArray` as a CoAP TCP `Message`:
+
+```js
+import kotlin from 'kotlin'
+import koapModule from '@juullabs/koap'
+
+const koap = koapModule.com.juul.koap
+
+let encoded = new Uint8Array(/* ...message data... */)
+let message = koap.decodeTcp(encoded)
+```
```

Comment:
Move below **Header** section? Otherwise it looks like Header is intended to be a Javascript example.